kratos/sources/kratos_parameters.cpp

```diff
@@ -954,7 +954,7 @@ void Parameters::Append(const Vector& rValue)
     nlohmann::json j_array(0.0, size);
 
     for (IndexType i = 0; i < size; ++i) {
-        j_array = rValue[i];
+        j_array[i] = rValue[i];
     }
```

Comment:
```suggestion
    j_array = rValue;
```

I think you can neglect the loop and can do just this.

Reply:
good catch!